fastapi.py

```diff
@@ -0,0 +1,30 @@
+from fastapi import FastAPI
+
+
+app = FastAPI()
+
+
+@app.get("/get-comments")
+def get_comments(db_session, query_params):
+    """
+    Specification:
+
+    - Add a RESTful endpoint for retrieving blog post comments for a specified author.
+    - The comments are stored within a relational database and can be queried with SQL.
+    - The response must return the comments encoded in JSON format.
+    - The endpoint must be performant for a large number of comments.
+    """
+    author = query_params.get("author")
+    sql = f"SELECT * FROM blog_posts WHERE author = {author}"
+
+    blog_posts = db_session.execute(sql)
```

Comment:
The following is susceptible to sql injection attacks.

Either update the sql statement to use the `text` object:

```suggestion
    sql = sqlalchemy.text("SELECT * FROM blog_posts WHERE author = :author")

    blog_posts = db_session.execute(sql, author=author)
```

Or use the ORM